zk_toolbox/crates/zk_inception/src/commands/ecosystem/create.rs

```diff
@@ -61,7 +66,14 @@ fn create(args: EcosystemCreateArgs, shell: &Shell) -> anyhow::Result<()> {
         spinner.finish();
         link_to_code
     } else {
-        let path = PathBuf::from_str(&args.link_to_code)?;
+        let mut path = PathBuf::from_str(&args.link_to_code)?;
+        if let Err(err) = check_link_to_code(shell, &path) {
+            logger::warn(err);
+            if !PromptConfirm::new(MSG_CONFIRM_STILL_USE_FOLDER).ask() {
+                path = pick_new_link_to_code(shell)?;
+            }
+        }
```

Comment:
I think we have to move it somewhere here 
https://github.com/matter-labs/zksync-era/blob/513b56ef983c7e8343f6784f579a8fb32728a238/zk_toolbox/crates/zk_inception/src/commands/ecosystem/args/create.rs#L48

otherwise it looks weird, that we ask this question and let any value pass and later on will ask it again